spaceship/lib/spaceship/two_step_client.rb

```diff
@@ -40,7 +40,16 @@ def handle_two_step(response)
         device_id = result.match(/.*\t.*\t\((.*)\)/)[1]
         select_device(r, device_id)
       elsif r.body.kind_of?(Hash) && r.body["trustedPhoneNumbers"].kind_of?(Array) && r.body["trustedPhoneNumbers"].first.kind_of?(Hash)
-        handle_two_factor(r)
+        puts("Two Step Verification for account '#{self.user}' is enabled")
+        puts("Please select a phone number to verify your identity")
+
+        available = r.body["trustedPhoneNumbers"].collect do |current|
+          "#{current['numberWithDialCode']}\tSMS\t(#{current['id']})"
+        end
+        result = choose(*available)
+        device_id = result.match(/.*\t.*\t\((.*)\)/)[1]
```

Comment:
can we rename this variable to `phone_id` as well? better to clarify that we are talking about phones that can receive sms here, not any devices (like tablets with no SIM etc)